README.md

```diff
@@ -29,17 +29,17 @@ There are two ways to update the resources:
 
 **Option 1** If you would like to use pheweb-native genes pre-processing step which includes downloading gencode gene annotation file, selecting protein-coding/IG/TR genes, de-duplicating gene symbols and gene ids, run the following:
 ```
-# downloads fresh genes annotation gtf and formats genes file that will be used in other steps
+# download fresh genes annotation gtf and formats genes file
 pheweb download-genes
 
-# adds nearest_genes column to the variant sites by using genes file generated above
+# add nearest_genes column to the variant sites by using genes file generated above
 pheweb add-genes
 
-# creates genes aliases db using genes file
+# create genes aliases db using freshly created genes file
 pheweb make-gene-aliases-sqlite3
 ``` 
 
-**Option 2** If you would like to keep gene types other than protein-coding/IG/TR in the pheweb broser, perform steps described below:
+**Option 2** If you would like to keep gene types other than protein-coding/IG/TR in pheweb browser, perform steps described below:
```

Comment:
Option 2 could be "Use a custom genes file".
The gene file needs to be bed file with build 38 coordinates where there are no duplicate genes.  Example below for generating file but preserving also other than protein coding genes.....
something like that